lib/features/email/presentation/controller/single_email_controller.dart

```diff
@@ -1465,7 +1479,8 @@ class SingleEmailController extends BaseController with AppLoaderMixin {
 
   void _handleParseCalendarEventSuccess(ParseCalendarEventSuccess success) {
     emailLoadedViewState.value = Right<Failure, Success>(success);
-    calendarEvent.value = success.calendarEventList.first;
+    calendarEvent.value = success.calendarEventMap.values.first.first;
+    _displayingEventBlobId = success.calendarEventMap.keys.first;
```

Comment:
Why not use `_displayingEventBlobId` as an element of `calendarEvent`

Reply:
Please elaborate.

Reply:
What do you create `_displayingEventBlobId` for?

Reply:
See https://github.com/linagora/tmail-flutter/pull/2803/commits/10fee68cc5befb6e4e13806cc8f836e740986e62#diff-cf11fcd6b3a412ebb11a6905f411746a62545237d976bc473b56bb9545472623R1704